omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/RetrospectiveVisitFragmentControllerTest.java

```diff
@@ -95,11 +95,10 @@ public void test_shouldCreateNewRetrospectiveVisit_whenNoStopDateSpecified() thr
 
         // should round to the time components to the start and end of the days, respectively
         Date expectedStartDate = new DateTime(2012, 1, 1, 0, 0, 0, 0).toDate();
-        Date expectedStopDate = new DateTime(2012, 1, 1, 23, 59, 59, 999).toDate();
```

Comment:
Why remove the above line?

Reply:
This test is to verify what I had changed in the sense that now we can allow the  user to pass a null end Date and not default it to the startDate